crates/iota-core/src/congestion_tracker.rs

```diff
@@ -18,9 +18,14 @@ use crate::execution_cache::TransactionCacheRead;
 /// Capacity of the congestion tracker's cache.
 const CONGESTION_TRACKER_CACHE_CAPACITY: u64 = 10_000;
 
-/// Alias type for holding transaction's gas price and mutable (or
-/// congested) shared objects.
-type TransactionGasPriceMutSharedObjectsPair = (u64, Vec<ObjectID>);
+/// Threshold for hotness below which an object is considered cold.
+const HOTNESS_CUTOFF: f64 = 1.0;
+
+/// Controls how quickly congestion feedback updates object hotness.
+const HOTNESS_ADJUSTMENT_FACTOR: f64 = 0.2; // > 0.0
+
+/// Controls how quickly hotness decays for objects not seen in congestion.
+const HOTNESS_DECAY_FACTOR: f64 = 1.5; // > 1.0 = decay, 1.0 = no decay
```

Comment:
Add accepted values to the doc string:

```suggestion
/// Controls how quickly hotness decays for objects not seen in congestion.
/// Values should be >= 1.0: set to > 1.0 for decay, or 1.0 for no decay.
const HOTNESS_DECAY_FACTOR: f64 = 1.5;
```